### ISSUE TITLE
Upgrading keras gives decimal values for true_positive, false_positive, true_negative, false_negative

### ISSUE THREAD

**Issue by amolgharat** [netrack/keras-metrics#50]
After upgrading Keras from version 2.2.5 to 2.3 I get decimal values for true_positive, false_positive, true_negative, false_negative. It works fine with version 2.2.5. Also, Keras version 2.3 has added new metrics like true_positive, false_positive, precision, recall etc. However, these metrics values don't match between Keras and keras-metrics.

**Comment by ybubnov** [comment]
@amolgharat, true, seems builds (and tests) are failing for Keras `2.3.0` *and tensorflow 2.0.0*.

**Comment by ybubnov** [comment]
I've added in #49 a deprecation warning for this library that support will be dropped since Keras `2.3.0` (as they already have a replacement for all metrics).

**Comment by monitarb** [comment]
Please son't stop the good job you were doing.
Keras Metrics in version 2.3.1 don´t work properly with to_categorical classes (i.e One hot encoded [0,1] instead of just 1).
False Positives and False Negatives give the same result (don't know why), so accuracy and recall give the same result as well.
I downgraded my keras version to 2.2.5 just so I could use your library that gives correct results.
Please, keep up the good work!
